test/core/event_engine/test_suite/cf_event_engine_test.cc

```diff
@@ -19,15 +19,22 @@
 
 #include "src/core/lib/event_engine/cf_engine/cf_engine.h"
 #include "test/core/event_engine/test_suite/event_engine_test.h"
+#include "test/core/event_engine/test_suite/oracle_event_engine_posix.h"
 #include "test/core/util/test_config.h"
 
 int main(int argc, char** argv) {
   testing::InitGoogleTest(&argc, argv);
   grpc::testing::TestEnvironment env(&argc, argv);
+  gpr_set_log_verbosity(GPR_LOG_SEVERITY_DEBUG);
+
   auto factory = []() {
     return std::make_unique<grpc_event_engine::experimental::CFEventEngine>();
   };
-  SetEventEngineFactories(factory, factory);
+  auto oracle_factory = []() {
+    return std::make_unique<
+        grpc_event_engine::experimental::PosixOracleEventEngine>();
```

Comment:
This may mean that the posix engine is used for the `Listener` implementation on whatever platform is running the iOS tests. Is that ok?

Reply:
Yes, client test needs a server which we don't have in cf_engine